docs/design/storagenode-graceful-exit.md

```diff
@@ -0,0 +1,89 @@
+- When a Storage node triggers graceful exit I want their allocated bandwidth on the network to be ignored so that they can complete their graceful exit as quickly as possible.
+- When a node dose not complete the graceful exit entirely I want the satellite to keep track of this so that they are subject to be DQed and their escrow kept
+	- this includes the node sending bad data to other nodes
+	- the node not transferring some pieces it holds
+- When a Storage Node leaves the network I want the ability to run a report on the satellite to get information about what Storage Nodes exited so that I can pay them their escrow amounts. 
+	- Create a report on the satellite to display which nodes have exited gracefully during a specified time frame. The report must include:
+		- NodeID
+		- Wallet address
+		- The date the node joined the network
+		- The date the node exited
+		- GB Transferred (amount of data the node transferred during exiting)		
+- When a Storage Node exits the network gracefully I want the satellite to have the ability to track how much egress they used for exiting so that we do not pay them for that bandwidth.
+- When A Storage Node is in the process of gracefully exiting the network I want them to be selected for download requests so that they can continue to contribute to the overall network.
+	- we need to be able to distinguish the bandwidth used for serving up data to clients vs bandwidth used for graceful exit. the node will NOT get paid for bandwidth used for graceful exit but will get paid for the bandwidth used to serve data to clients.
+- When a Storage Node is in the process of gracefully exiting the network I want them to continue to be audited and their uptime checked so we can ensure they are still "good" nodes.
+
+
+## Business Requirements/ Job Stories (Non MVP)
+
+- When a Storage Node completes graceful exit for all satellites I want their Node to automatically shut down so that they do not have to worry about receiving any more data.
+- When a Storage node no longer wants to store data for a satellite I want them to have the ability to run graceful exit for a specific satellite so that they do not lose their escrow on that satellite. 
+- When a Storage Node wants to rejoin a satellite they previously exited gracefully I want them to have a simple command they can run so that the satellite is notified and they can start being selected for storage in the node selection process.
+	- The node will restart the escrow process if they decide to rejoin a satellite. 
+- When a Storage Node operator wants to reduce the amount of storage space they have allocated to the network I want them to have the ability to do a “partial graceful exit which would transfer some of the data they currently have onto other nodes so they have a way of reducing their storage allocation without just deleting data and failing audits. 
```

Comment:
Let's clarify that the decision which pieces will be moved to other storage nodes will be on the satellite, not on the exiting storage nodes.

Reply:
added clarification